appium-cookies/lib/cookies.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * derived from jQuery Cookie Plugin v1.4.1
+ * https://github.com/carhartl/jquery-cookies
+ */
+ //needed to communicate between JSON protocol cookies and regular js cookies 
+ // doesn't actually store the cookies kind of just translates them 
+import _ from 'underscore';
```

Comment:
I think we are switching to [lodash](https://lodash.com/) over underscore. For the most part this just means changing the import to `import _ from 'lodash'` and running `npm install --save lodash`.